service/internal/images/images.go

```diff
@@ -117,6 +117,26 @@ func (i *Images) Versions(ctx context.Context, obj interface{}) (k8scloudconfig.
 	return v, nil
 }
 
+func (i *Images) AWSCNI(ctx context.Context, obj interface{}) (string, error) {
```

Comment:
Can we please order the methods alphabetically? Above is `Version`. This here is `AWSCNI`.

Reply:
done